plugins/src/android/OptimoveInitProvider.java

```diff
@@ -10,12 +10,17 @@
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+
 import com.optimove.android.Optimove;
 import com.optimove.android.OptimoveConfig;
+import com.optimove.android.optimobile.OptimoveInApp;
 
 public class OptimoveInitProvider extends ContentProvider {
     private static final String OPTIMOVE_CREDENTIALS = "optimoveCredentials";
     private static final String OPTIMOVE_MOBILE_CREDENTIALS = "optimoveMobileCredentials";
+    private static final String IN_APP_AUTO_ENROLL = "auto-enroll";
+    private static final String IN_APP_EXPLICIT_BY_USER = "explicit-by-user";
+    private static final String KEY_IN_APP_CONSENT_STRATEGY = "inAppConsentStrategy";
```

Comment:
Structure: can we group the keys together / common prefixes if they're all keys? (i.e. OPTIMOVE_CREDENTIALS -> KEY_...)

Reply:
sure thing